diff --git a/luci-app-ua2f/Makefile b/luci-app-ua2f/Makefile
--- a/luci-app-ua2f/Makefile
+++ b/luci-app-ua2f/Makefile
@@ -2,7 +2,7 @@
 
 PKG_NAME:=luci-app-ua2f
 PKG_VERSION=1.8.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
 
@@ -12,7 +12,7 @@
 	SECTION:=luci
 	CATEGORY:=LuCI
 	SUBMENU:=3. Applications
-	DEPENDS:=+ua2f
+	DEPENDS:=+ua2f +luci-compat
 	TITLE:=file transfer tool
 	PKGARCH:=all
 endef
diff --git a/luci-app-ua2f/controller/autoua2f.lua b/luci-app-ua2f/controller/autoua2f.lua
--- a/luci-app-ua2f/controller/autoua2f.lua
+++ b/luci-app-ua2f/controller/autoua2f.lua
@@ -1,7 +1,7 @@
 module("luci.controller.autoua2f", package.seeall)
 
 function index()
-    entry({"admin", "services", "autoua2f"}, cbi("autoua2f"), _("防检测"), 94)
+    entry({"admin", "services", "autoua2f"}, cbi("autoua2f"), _("UA2F"), 94)
     entry({"admin", "services", "autoua2f", "status"}, call("act_status")).leaf = true
 
 end
diff --git a/luci-app-ua2f/model/cbi/autoua2f.lua b/luci-app-ua2f/model/cbi/autoua2f.lua
--- a/luci-app-ua2f/model/cbi/autoua2f.lua
+++ b/luci-app-ua2f/model/cbi/autoua2f.lua
@@ -1,10 +1,7 @@
 m = Map("autoua2f", translate("UA2F"))
-m.description = translate([[
-        <span style="font-family: '微软雅黑'; color: red">该插件并非适合所有类型的检测！！！有网络的情况时下方会显示你的UA，如果两个UA不一样就说明成功了</span>
-    ]])
+m.description = translate("配置 UA2F 服务。默认关闭；保存并应用后会更新配置及服务状态。修改 User-Agent 不保证消除认证平台提示。")
     
 m:section(SimpleSection).template = "ua2f/ua2f"
-m:section(SimpleSection).template = "ua2f/ua2f_A"
 
 e = m:section(TypedSection, "autoua2f", translate(""))
 e.addremove = false
@@ -27,7 +24,7 @@
 o5.rmempty = false
 
 o6 = e:option(Value, "Custom_UA", translate("自定义用户代理"), translate("自定义用户代理字符串，长度不足则填充空格，过长则截取与原来长度相同的子串"))
-o6.default = "Mozilla/5.0 (Window NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/555.66"
+o6.default = ""
 
 m.on_commit = function(self)
     luci.sys.call("/etc/init.d/autoua2f start")
